benchmarks/cpp/tensorexpr/bench_ops.py

```diff
@@ -0,0 +1,56 @@
+import timeit
+import torch
+
+torch._C._jit_override_can_fuse_on_cpu(True)
+torch._C._debug_set_fusion_group_inlining(False)
+
+def hardswish(x):
+    return x * torch.clamp(x + 3., 0., 6.) / 6.
+
+
+unary_ops = [
+    hardswish,
+    torch._C._nn.hardswish,
+    torch.sigmoid,
+    torch.reciprocal,
+    torch.neg,
+    torch.relu,
+    torch.isnan,
+    torch.log,
+    torch.log10,
+    torch.log1p,
+    torch.log2,
+    torch.exp,
+    torch.expm1,
+    torch.erf,
+    torch.erfc,
+    torch.cos,
+    torch.sin,
+    torch.tan,
+    torch.acos,
+    torch.asin,
+    torch.cosh,
+    torch.sinh,
+    torch.atan,
+    torch.tanh,
+    torch.sqrt,
+    torch.rsqrt,
+    torch.abs,
+    torch.ceil,
+    torch.floor,
+    torch.round,
+    torch.trunc,
+    torch.lgamma,
+]
+
+print("{:20s} {:>10s} {:>10s} {:>10s}".format(
+    "op", "eager", "nnc", "speedup"))
+
+for op in unary_ops:
+    x = torch.rand((1024, 1024))
+    traced = torch.jit.trace(lambda x: op(x), (x))
+    [traced(x) for _ in range(2)]
+    torch.testing.assert_allclose(op(x), traced(x))
+    teager = timeit.timeit(stmt="op(x)", globals=globals(), number=100)
```

Comment:
Minor: maybe a warmup to be more faithful? 

Also disable the multi-threading in PyTorch from the benchmark with torch.set_num_threads(1)?

Reply:
Heh, I always nuke threading from orbit with `numactl` or `taskset` but yeah good idea.

Reply:
Oh, and warmup is the `[traced(x) for _ in range(2)]` above, but it's easy to overlook :-p